Makefile

```diff
@@ -1,5 +1,10 @@
-GO_VERSION=1.9
-GO_FILES=$(shell find . -type f -name "*.go")
+GO_FILES := $(shell find . -type f -name "*.go")
+GO_BUILD := CGO_ENABLED=0 go build -ldflags="-w -s"
```

Comment:
This is not used anymore, or?

Reply:
Oh good catch. It should be in the build targets. I'll fix it.

Reply:
pushed a fixup